### DISCUSSION TITLE
Allow pinch zooming in editor

### DISCUSSION THREAD

**Discussion by damionx7** [damionx7/Zettel-Notes-Documentation#145] [Feature Requests]
Suggested by [Erik]

**Comment by nathanl** [comment]
For me, pinch zooming resizes the text, which I think is perfect. I actually don't like how mobile browsers often "zoom in" with this gesture. I don't want to take a close look at the serif on a letter nearly as often as I want to resize the text overall and keep it wrapping as it should.
